fvm/src/syscalls/crypto.rs

```diff
@@ -47,6 +48,50 @@ pub fn verify_signature(
         .map(|v| if v { 0 } else { -1 })
 }
 
+pub fn verify_bls_aggregate(
+    context: Context<'_, impl Kernel>,
+    num_signers: u32,
+    sig_off: u32,
+    pub_keys_off: u32,
+    digests_off: u32,
+) -> Result<i32> {
+    let pub_keys_len = num_signers * BLS_PUB_LEN as u32;
+    let digests_len = num_signers * BLS_DIGEST_LEN as u32;
```

Comment:
We need to be careful about integer overflows here.

Reply:
Overflowing would mean 44739242 signatures are being aggregated, I'm not sure if that's expected. But, I can add some `checked_mul`s.

Reply:
It's not expected, but that's a parameter from a potentially untrusted user so we can't assume anything.